tools/cldr-apps/src/main/webapp/js/CldrSurveyVettingLoader.js

```diff
@@ -1591,15 +1621,15 @@ function showV() {
 													}
 
 													var saveList = {
-														locale: surveyCurrentLocale,
+														locale: cldrStatus.getCurrentLocale(),
 														confirmList: confirmList,
 													};
 
 													console.log(saveList.toString());
 													console.log("Submitting " + type + " " + confirmList.length + " for confirm");
-
-													var url = cldrStatus.getContextPath() + "/SurveyAjax?what=oldvotes&_=" + surveyCurrentLocale + "&s=" + surveySessionId + "&doSubmit=true&" + cacheKill();
-													myLoad(url, "(submitting oldvotes " + surveyCurrentLocale + ")", function(json) {
+													const curLocale = cldrStatus.getCurrentLocale();
+													var url = cldrStatus.getContextPath() + "/SurveyAjax?what=oldvotes&_=" + curLocale + "&s=" + cldrStatus.getSessionId() + "&doSubmit=true&" + cacheKill();
+													myLoad(url, "(submitting oldvotes " + curLocale + ")", function(json) {
```

Comment:
too deeply indented!!! should have 4 spaces instead tab (which github displays as 8 spaces) -- but in any case, too deeply indented